src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectConverter.cs

```diff
@@ -111,4 +111,29 @@ internal override void WriteAsPropertyNameCore(Utf8JsonWriter writer, object? va
             runtimeConverter.WriteAsPropertyNameCoreAsObject(writer, value, options, isWritingExtensionDataProperty);
         }
     }
+
+    /// <summary>
+    /// A placeholder ObjectConverter used for driving boxed root value serialization only
```

Comment:
(nit) Is "boxed" really the right term here? To me, as a .NET dev, I think of value types being boxed. But that isn't always the case here. It could be a regular "class" being serialized "as object".